sdk/netapp/azure-mgmt-netapp/tests/conftest.py

```diff
@@ -49,7 +49,7 @@ def add_sanitizers(test_proxy):
     add_general_regex_sanitizer(regex=subscription_id, value="00000000-0000-0000-0000-000000000000")
     add_general_regex_sanitizer(regex=tenant_id, value="00000000-0000-0000-0000-000000000000")
     add_general_regex_sanitizer(regex=client_id, value="00000000-0000-0000-0000-000000000000")
-    add_general_regex_sanitizer(regex=client_secret, value="00000000-0000-0000-0000-000000000000")
+    add_general_string_sanitizer(value="00000000-0000-0000-0000-000000000000", target=client_secret)
```

Comment:
Please revert this line since it doesn't work: https://github.com/Azure/azure-sdk-for-python/pull/26978/files#r1003922325

Reply:
will change to `urllib.parse.quote(client_secret)`  that seems to work